README.md

```diff
@@ -91,3 +91,7 @@ The `DEBUG` flag does all sorts of magical things, to the point where testing wi
 - Django bypasses the `ALLOWED_HOST` restrictions, which again can lead to `400 Bad Request` errors in `DEBUG=False` setting.
 - Rather than HTTP error pages, Django will generate stack traces pages that expose pretty much all environment variables except any that match certain substrings such as `KEY`, `PASS`, etc. for obvious security reasons.
 - ...there are probably more gotchas just for `DEBUG` so if you find any please add them to this list.
+
+## Contributing
+
+We love contributors, but the team maintaining this project is small and not structured to significantly support new and inexperienced ones. If there's an unassigned issue that catches your eye, you're free to open a PR for it but keep in mind our support will be limited. We don't have the capacity to walk you through the process of spinning up the project, opening a PR or describing what the solution to the issue could be.
```

Comment:
`inexperienced ones` => `inexperienced contributors`
`you're free to open` => `feel free to open`
`for it but` => `for it, but`
`We don't have` => `We usually don't have` (because we _might_ from time to time depending on the issue)

Reply:
Wanna inline suggest those?

Reply:
for a full test paragraph, inline suggestions are nowhere near as easy to write as code suggestions =(

Reply:
```suggestion
We love contributors, but the team maintaining this project is small and not structured to significantly support new and inexperienced contributors. If there's an unassigned issue that catches your eye, feel free to open a PR for it, but keep in mind our support will be limited. We usually don't have the capacity to walk you through the process of spinning up the project, opening a PR or describing what the solution to the issue could be.
```

Reply:
@Pomax what? 😛